tests/manage/pv_services/pv_encryption/test_encryptedrbd_pvc_status_override_vaultnm_backendpath_tenant_configmap.py

```diff
@@ -0,0 +1,182 @@
+        self,
+        kv_version,
+        use_vault_namespace,
+        pv_encryption_kms_setup_factory,
+        project_factory,
+        storageclass_factory,
+        pod_factory,
+    ):
+        """
+        Setup csi-kms-connection-details configmap
+
+        """
+
+        log.info("Setting up csi-kms-connection-details configmap")
+        self.kms = pv_encryption_kms_setup_factory(kv_version, use_vault_namespace)
+        log.info("csi-kms-connection-details setup successful")
+
+        # Create a project
+        self.proj_obj = project_factory()
+
+        # Create an encryption enabled storage class for RBD
+        self.sc_obj = storageclass_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            encrypted=True,
+            encryption_kms_id=self.kms.kmsid,
+        )
+
+    def test_encryptedrbd_pvc_status_with_tenant_configmap_override(
+        self, setup, multi_pvc_factory, kv_version, kms_provider, pod_factory, use_vault_namespace
+    ):
+
+        vault_resource_name = create_unique_resource_name("test", "vault")
+        if use_vault_namespace:
+            self.kms.vault_create_namespace(namespace=vault_resource_name)
+        self.kms.vault_create_backend_path(
+            backend_path=vault_resource_name, kv_version=kv_version
+        )
+        self.kms.vault_create_policy(policy_name=vault_resource_name)
+
+        # Create a configmap in the tenant namespace to override the vault namespace as shown below:
+        if use_vault_namespace:
+            self.kms.create_tenant_configmap(
+                self.proj_obj.namespace, vaultBackendPath=f"{vault_resource_name}",
+                vaultNamespace=f"{self.kms.vault_namespace}"
+            )
+        else:
+            self.kms.create_tenant_configmap(
+                self.proj_obj.namespace, vaultBackendPath=f"{vault_resource_name}",
+            )
+
+        # Create ceph-csi-kms-token in the tenant namespace
+        self.kms.vault_path_token = self.kms.generate_vault_token()
+        self.kms.create_vault_csi_kms_token(namespace=self.proj_obj.namespace)
+
+        # Create New PVC and check status
+        self.pvc_size = 1
+        self.pvc_obj = multi_pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            project=self.proj_obj,
+            storageclass=self.sc_obj,
+            access_modes=[
+                f"{constants.ACCESS_MODE_RWX}-Block",
+                f"{constants.ACCESS_MODE_RWO}-Block",
+            ],
+            size=self.pvc_size,
+            status=constants.STATUS_BOUND,
+            num_of_pvc=2,
+            wait_each=False,
+        )
+
+        # Verify if the key is created in Vault
+        self.vol_handles = []
+        for pvc_obj in self.pvc_obj:
+            pv_obj = pvc_obj.backed_pv_obj
+            vol_handle = pv_obj.get().get("spec").get("csi").get("volumeHandle")
+            self.vol_handles.append(vol_handle)
+
+            if kms_provider == constants.VAULT_KMS_PROVIDER:
```

Comment:
Given how the test is written, this if statement will always be true. There is no need for this check